test/models/member_test.rb

```diff
@@ -175,4 +175,15 @@ class MemberTest < ActiveSupport::TestCase
 
     assert_nil member.last_membership
   end
+
+  test "schedules a job for updating Neon membership when enabled" do
+    member = create(:member)
+
+    NeonMemberJob.stub :perform_async, true do
+      member.stub :can_update_neon_crm?, true do
+        member.update(city: "#{member.city} Test")
+        assert_send([NeonMemberJob, :perform_async])
```

Comment:
I think that `assert_send` asserts the result of calling the passed method, but in this case what we probably want is to assert that the `perform_async` method was already called by other code (in this case, the callback within Member).

To do that, we'd need a mock instead of a stub (at least in Minitest): https://semaphoreci.com/community/tutorials/mocking-in-ruby-with-minitest#h-mocking

Reply:
Makes sense! I think the update I just pushed should fix it